example_data/corings.csv

```diff
@@ -1,3 +1,4 @@
-wkt_geom;ID;loc_id;xcoord;ycoord;zcoorddb
-Point (32485574.71000000089406967 5949969.473000000230968);Core 1;1227;32485574.71;5949969.473;1.4596
-Point (32485583.17999999970197678 5950059.70899999979883432);Core 2;1226;32485583.18;5950059.709;1.3083
+ID;loc_id;xcoord;ycoord;zcoorddb
```

Comment:
Did you remove the column **wkt_geom** on purpose?

Reply:
It's unnecessary, thats why I removed it ( ... loc_id would be unnecessary, too). But it won't be a problem to let it in the file.